scripts/obtain/dotnet-install.ps1

```diff
@@ -173,23 +173,23 @@ function GetHTTPResponse([Uri] $Uri)
 }
 
 
-function Get-Latest-Version-Info([string]$AzureFeed, [string]$AzureChannel, [string]$CLIArchitecture) {
+function Get-Latest-Version-Info([string]$AzureFeed, [string]$Channel, [string]$CLIArchitecture) {
```

Comment:
The runtime has a download for Linux-arm:
https://dotnetcli.blob.core.windows.net/dotnet/Runtime/2.1.0-preview1-25322-04/dotnet-linux-arm.2.1.0-preview1-25322-04.tar.gz